CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 :wrench: **Fixes**
 
 - Images - Add `size` and `srcset` attributes to the images component for responsive loading of images. ([Issue 422](https://github.com/nhsuk/nhsuk-frontend/issues/422))
+- Pagnation - Add `padding-left` and `padding-right` attributes to the pagination class `nhsuk-pagination__link--prev` `nhsuk-pagination__link--next`. ([Issue 448](https://github.com/nhsuk/nhsuk-frontend/issues/448))
```

Comment:
Change log entries usually describe the problem that has been fixed, rather than the implementation.

So something like: 

> Pagination - Add padding to the pagination next and previous links to prevent the box-shadow overlapping the other link on hover. ([Issue 448](https://github.com/nhsuk/nhsuk-frontend/issues/448))